e2e/terraform/README.md

```diff
@@ -101,9 +101,9 @@ PATH="$HOME/bin:$HOME/.local/bin:/usr/local/bin:$PATH"
 $ source ~/.profile
 ```
 
-## Creating the Config For Cluster
+## Creating the Configuration for Cluster
```

Comment:
Creating the Configuration for Cluster = Creating the Configuration File for Cluster

Reply:
Done